spinta/manifests/tabular/helpers.py

```diff
@@ -1869,7 +1870,7 @@ def _read_xlsx_manifest(path: str) -> Iterator[Tuple[str, List[str]]]:
 
         empty_rows = _empty_rows_counter()
         for i, row in enumerate(rows, 2):
-            row = [row[c] if c is not None else None for c in cols]
+            row = [row[c] or "" if c is not None else None for c in cols]
```

Comment:
kokiais atvejais čia turėtų gautis kabutės?

Reply:
Kadangi `c` yra stulpelio numeris tai labiau priklauso ne nuo `c` reiksmes o nuo to kas yra `row` masyve. Kabutes gaunasi kai eiltute (`row`) turime `None` elementu. Kitais zodizias šioje vietoje unifikuojama non-value iš None keičiama į "", kad kitose vietose išvengti kai `None` tampa `"None"`

![image](https://github.com/user-attachments/assets/fd3799e8-c331-480e-8763-a440e6205222)

Reply:
bet, kaip suprantu, `None` vis tiek gali būti tuo atveju, jei pats `c` yra `None`?